build.psm1

```diff
@@ -253,12 +264,25 @@ function Start-ScriptAnalyzerBuild
         # The Rules project has a dependency on the Engine therefore just building the Rules project is enough
         try {
             Push-Location $projectRoot/Rules
-            Write-Progress "Building ScriptAnalyzer for PSVersion '$PSVersion' using framework '$framework' and configuration '$Configuration'"
+            $message = "Building ScriptAnalyzer for PSVersion '$PSVersion' using framework '$framework' and configuration '$Configuration'"
+            Write-Verbose -Verbose:$verboseWanted -Message "$message"
+            Write-Progress "$message"
             if ( -not $script:DotnetExe ) {
                 $script:DotnetExe = Get-DotnetExe
             }
-            $buildOutput = & $script:DotnetExe build --framework $framework --configuration "$buildConfiguration" 2>&1
+            $dotnetArgs = "build",
+                "--framework",
+                $framework,
+                "--configuration",
+                "$buildConfiguration"
+            if ( $env:InVSTS ) {
```

Comment:
The `TF_BUILD` environment variable is defined in Azure DevOps and could be used for that instead of defining a custom variable on the release build. Or do you need to specifically know if you are running in the release build? If so, why do we need to build differently with a different output directory when we copy it to the module directory anyway at the end?

Reply:
I'll try that - thanks!

Reply:
The more important question to understand for me is why we cannot keep the existing directory structure?